apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/preadj/McsClaimResponseTransformerV2.java

```diff
@@ -96,11 +97,22 @@ private static ClaimResponse transformClaim(PreAdjMcsClaim claimGroup) {
     claim.setContained(List.of(getContainedPatient(claimGroup)));
     claim.setExtension(getExtension(claimGroup));
     claim.setIdentifier(getIdentifier(claimGroup));
-    claim.setStatus(
```

Comment:
Would this be a good use case for Optional?
```
String statusCode = Optional.ofNullable(claimGroup.getIdrStatusCode()).orElse("");
```

Reply:
It would be syntactically terser, I thought going with an `if()` might make it easier to understand there is an edge case we are resolving here.

Reply:
Rather than Optional for a string value you can use `Strings.nullToEmpty()`

```
String statusCode = Strings.nullToEmpty(claimGroup.getIdrStatusCode());
```